### PR TITLE
Add Xiaomi Miio Plug config flow

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add config flow to xiaomi miio plug platform.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46179
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -1706,7 +1706,11 @@ Turn the eyecare mode off.
 
 The `xiaomi_miio` switch platform allows you to control the state of your Xiaomi Smart WiFi Socket aka Plug, Xiaomi Smart Power Strip and Xiaomi Chuangmi Plug V1.
 
-Please follow the instructions on [Retrieving the Access Token](/integrations/xiaomi_miio/#retrieving-the-access-token) to get the API token to use in the `configuration.yaml` file.
+Please follow the instructions on [Retrieving the Access Token](/integrations/xiaomi_miio/#retrieving-the-access-token) to get the API token to use during configuration flow setup.
+
+### Configuration flow setup
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown]:
```suggestion
### Configuration
```

File: source/_integrations/xiaomi_miio.markdown [hunk 2]
```diff
@@ -1706,7 +1706,11 @@ Turn the eyecare mode off.
...
+
+To add a plug to your installation, click Configuration in the sidebar, then click Integrations and then click the + icon in the lower right and find xiaomi_miio. Select the option “Connect to a other Xiaomi Miio device” and click submit. You will then be presented with a form in which you will need to fill in the “IP address” and 32 characters “token”. Optionally, you can specify a different name for the plug. After you click submit, you will have the opportunity to select the area that your devices are located.
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown]:
```suggestion
To add a plug to your installation, click Configuration in the sidebar, then click Integrations and then click the + icon in the lower right and find xiaomi_miio. Select the option “Connect to an other Xiaomi Miio device” and click submit. You will then be presented with a form in which you will need to fill in the “IP address” and 32 characters “token”. Optionally, you can specify a different name for the plug. After you click submit, you will have the opportunity to select the area that your devices are located.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by starkillerOG** [comment]
@frenck Sorry, I corrected some steps of the config flow that were removed.
I just realized it now.
Could you review again?

**Review by frenck — APPROVED** [review]
For now, let's keep the configuration like this. When all is migrated to a config flow, we should look into centralizing the configuration part in a single chapter.

**Pull request merged**